### ISSUE TITLE
Extension has stopped working

### ISSUE THREAD

**Issue by JHarrisGTI** [hediet/vscode-tasks-statusbar#11]
Hello! I've used and loved this extension for several years--since I started using VSCode, I think. Now it has stopped working and I don't know why. 😭 Buttons no longer appear on the status bar, no matter what I put in the Task Label Filter. I'm not seeing any error messages, so I don't know where to start troubleshooting. Is there anything I can do to revive my status bar buttons?

**Comment by hediet** [comment]
Can you try a previous version of VS Code? It stills works for me and I use it daily ;)

**Comment by JHarrisGTI** [comment]
Thanks for getting back to me. I tried a previous version of VSCode; that made no difference. So I tried creating a new profile in VSCode and that fixed it. Eventually I narrowed it down to the UI state and figured out that somehow I'd disabled the taskbar buttons in the taskbar's context menu. Right-clicking on the taskbar and checking the "Tasks Statusbar (extension)" option fixed the problem. Sorry to bother you!
